elasticsearch/Dockerfile.origin

```diff
@@ -25,10 +25,12 @@ ENV ES_PATH_CONF=/etc/elasticsearch/ \
 
 ARG MAVEN_REPO_URL=https://repo1.maven.org/maven2/
 
+RUN rm /etc/yum.repos.d/*
+ADD art.repo /etc/yum.repos.d
+
 RUN  yum install -y --setopt=tsflags=nodocs --nogpgcheck \
          https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
      packages="java-${JAVA_VER}-openjdk-headless \
-              PyYAML  \
```

Comment:
why are we removing this?

Reply:
This is a fat-finger delete. The intention is to replace with `python36 python3-pyyaml` as provided by `UBI8:latest`.